.github/workflows/main.yml

```diff
@@ -320,4 +347,4 @@ jobs:
       - name: Build
         run: mvn -B package -DskipTests=true
       - name: Shortly run DuckDB
-        run: cd target && java -jar $(ls | grep -P 'sqlancer-[0-9.]*.jar') --num-threads 4 --timeout-seconds 30 --num-queries 0 duckdb
+        run: cd target && java -jar $(ls | grep -P 'sqlancer-[0-9.]*.jar') --num-threads 4 --timeout-seconds 30 --num-queries 0 duckdb
```

Comment:
There are many changes where the newline of a file is removed. Could you please revert these?